### PR TITLE
KOGITO-8443 Added note about simplified format for Knative service discovery

### PR DESCRIPTION
**Pull request by hbelmiro**
**JIRA:** https://issues.redhat.com/browse/KOGITO-8443

Related to: https://github.com/kiegroup/kogito-runtimes/pull/2924

----------

- [x] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [x] The nav.adoc file has a link to this guide in the proper category
- [x] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Review by kaldesai — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/cloud/kubernetes-service-discovery.adoc [hunk 1]
```diff
@@ -30,6 +30,9 @@ The following scheme values are supported in the URI pattern:
 * `openshift`
 * `knative`
 
+> **NOTE**: When using `knative`, you can also use a simplified URI like:
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/cloud/kubernetes-service-discovery.adoc:33]: Wouldn't that conflict with https://github.com/kiegroup/kogito-docs/pull/325 ?

Comment (radtriste) [reply serverlessworkflow/modules/ROOT/pages/cloud/kubernetes-service-discovery.adoc:33]: nevermind I mixed up discovery and custom function

**Review by radtriste — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/cloud/kubernetes-service-discovery.adoc (hunk 1 above)
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/cloud/kubernetes-service-discovery.adoc:33]: on a second thought, I would have put that one after the listing of resources and mentioning it is looking directly for `serving.knative.dev/v1/service` resource

**Review by fjtirado — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/cloud/kubernetes-service-discovery.adoc (hunk 1 above)
Comment (hbelmiro) [reply serverlessworkflow/modules/ROOT/pages/cloud/kubernetes-service-discovery.adoc]: Done.
Also, I removed the warning about GVK being mandatory.

**Review by MarianMacik — APPROVED** [review]

**Review by radtriste — APPROVED** [review]

**Pull request merged**